README.md

```diff
@@ -1,6 +1,177 @@
 # OpenSlides authentication service
 
-Service for OpenSlides which handles the authentication of users.
+Service for OpenSlides which handles the authentication of users. After a successfully authentication an access_token (passed as header `authentication`) and a cookie (with name `refreshId`) is issued (in the following called `Ticket`).
+
+With this Ticket all routes, that are prefixed with a `secure`, are available.
+
+The combination of cookie and token protects against XSS- and CSRF-attacks. Such cookies is issued with the property `httpOnly`. To understand why and read further things, the document [Implement an auth-service](res/Ausarbeitung_Endfassung.pdf) can be read.
+
+## API
+
+### Interfaces
+
+```ts
+// The properties of this interface have to be passed as HTTP-headers in a request. 
```

Comment:
I miss some more attributes for tickets:
- A secured route always needs a ticket. Maybe remove `ticker: Ticket` from the arguments for secured routes, since it is implied, that secure routes always need a ticket
- If there is an invalid ticket, secured routes will internally do `authenticate`:
  - if it fails, a 405 forbidden will be returned (with a nice message...)
  - if it succeeds, a possible new access token will be set.